test/integration-new/run.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# This script runs integration tests on the EKS Amazon VPC CNI K8s
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+echo "Running VPC CNI K8s integration test with the following variables
+KUBE CONFIG: $KUBE_CONFIG_PATH
```

Comment:
Maybe we can pass these as inputs to the script? And maybe as a follow up, we can just pick from the current kube context if nothing is passed.